experimental/cloudstrapper/playbooks/devops-configure.yaml

```diff
@@ -5,7 +5,8 @@
     - {role: ansible-server, become: yes}
     - golang-host
     - build-platform
-    - {role: key-manager, tags: keyManager}
+    - {role: key-manager, tags: keyManager, become: true}
+    - clean-authorized-keys
```

Comment:
clean-authorized-keys appears to clear up the two keys that were created as part of the Cloudstrapper, but it is likely that customers might need this even if they cleanup the current deployment. Would it be good to club it under a "Never" tag so that people don't accidentally clean it up?

Reply:
Just noticed that this clears out the keys inside the devops instance which is perfect. The comment should be applicable to the cleanup of keys in the "cleanup" task.

Reply:
I have added never tag to the key cleanup of the cleanup playbook!